modules/vectra.py

```diff
@@ -63,6 +65,29 @@ def api_validator(self, *args, **kwargs):
 
     return api_validator
 
+def renew_access_token(func):
+    def wrapper(self, *args, **kwargs):
+        try:
+            return func(self, *args, **kwargs)
+        except HTTPUnauthorizedException:
```

Comment:
This exception should eventually give up.  It's possible refresh_oauth_token may fail.  Do we need to check the _refresh_oauth_token's result here, and raise (fail) if we are unable to refresh the token?